appveyor.yml

```diff
@@ -0,0 +1,53 @@
+version: "{branch}-ci-{build}"
+os:
+- Visual Studio 2013
```

Comment:
Ok - we need VS2013 because we have precompiled boost libs only for that version (C:\libraries\boost\lib-win32-vs12 (something like that))